opsmngr/oplog_store_config.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2029 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opsmngr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const backupAdministratorOplogBasePath = "admin/backup/oplog/mongoConfigs"
+
+// OplogStoreConfigService is an interface for using the Oplog
+// endpoints of the MongoDB Ops Manager API.
+//
+// See more: https://docs.opsmanager.mongodb.com/current/reference/api/admin/backup/oplog-store-config/
+type OplogStoreConfigService interface {
+	List(context.Context, *atlas.ListOptions) (*BackupStores, *atlas.Response, error)
+	Get(context.Context, string) (*BackupStore, *atlas.Response, error)
+	Create(context.Context, *BackupStore) (*BackupStore, *atlas.Response, error)
+	Update(context.Context, string, *BackupStore) (*BackupStore, *atlas.Response, error)
+	Delete(context.Context, string) (*atlas.Response, error)
+}
+
+// BackupConfigsServiceOp provides an implementation of the BackupConfigsService interface
+type OplogStoreConfigServiceOp service
+
+var _ OplogStoreConfigService = &OplogStoreConfigServiceOp{}
+
+// Get retrieves a Oplog.
+//
+// See more: https://docs.opsmanager.mongodb.com/current/reference/api/admin/backup/oplog/mongoConfigs/get-one-oplog-configuration-by-id/
+func (s *OplogStoreConfigServiceOp) Get(ctx context.Context, oplog string) (*BackupStore, *atlas.Response, error) {
```

Comment:
in the update you call it `oplogID` and here is just `oplog` let's keep it consistent and my vote is on `oplogID`